apps/meteor/client/lib/utils/isOverLicenseLimits.ts

```diff
@@ -0,0 +1,22 @@
+import type { LicenseLimitKind } from '@rocket.chat/license';
+
+type Limits = Record<
```

Comment:
OOS but this same type is used on the response of `licenses.info` endpoint, can we unify them?